_rules/SC1-1-1-accessible-name-is-image-filename.md

```diff
@@ -0,0 +1,89 @@
+---
+name: image accessible name is image filename
+description: |
+  This rule checks that `img` elements that use their source filename as their accessible name, do so without loss of infomation to the user.
```

Comment:
It's inconsistent with the applicability (just `img`). 

Proposal, replace 'ìmg`elements by "image" (as it is done in #251) :  

>   This rule checks that image that use their source filename as their accessible name, do so without loss of infomation to the user.